### DISCUSSION TITLE
Direct Ip Public

### DISCUSSION THREAD

**Discussion by ma-ramezani** [apache/cloudstack#9688] [Q&A] [answered]
how to assign public ip to vm in v4.18.2.3 without NAT or Forward?

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by weizhouapache** [comment, marked as answer]
You can use shared network

**Comment by pcfriek1987** [comment]
Or do so with VLANS in an advanced network.
